modules/executive/components/VoteModal/DefaultView.tsx

```diff
@@ -157,6 +158,19 @@ export default function DefaultVoteModalView({
         mutateVotedProposals();
         mutateMkrOnHat();
         onTransactionMined();
+
+        // Invalidate supporters cache
+        setTimeout(() => {
+          fetchJson(`/api/cache/invalidate?network=${network}`, {
+            method: 'POST',
+            headers: {
+              'Content-Type': 'application/json'
+            },
+            body: JSON.stringify({
+              cacheKey: executiveSupportersCacheKey
+            })
+          });
+        }, 30000);
```

Comment:
Could be useful to create helpers for these invalidation calls so that we can easily reuse and refactor